PhotoMapTests/Mocks/Services/FirestoreServiceMock.swift

```diff
@@ -0,0 +1,45 @@
+//
```

Comment:
Please add tests for:
- TimelineCoordinator
- TimelineViewModel outputs (getMarker, getTitle, etc)

We need to cover every input/output in ViewModel and Coordinator as well. This is a business logic